cmd/replay/main.go

```diff
@@ -46,6 +47,19 @@ func getBlockHeaderByHash(hash common.Hash) *types.Header {
 	return header
 }
 
+func getHotShotHeaderByHash(hash common.Hash) *espresso.Header {
```

Comment:
2. The replay executable is modified to be able to fetch a hotshot header preimage by hash, so that it can be passed into the STF.